CRC.adoc

```diff
@@ -1,80 +1,271 @@
 
-Scroll down to the *Repository Visibility* section, and ensure the repository is set to public:
+[source,bash]
+----
+crc setup
+----
 
-image::quay_repo_visibility.png[align=center]
+Also, in order to run the tests seamlessly and without any problems, there is a set of settings you are advised to
+change:
 
-7. Start the CRC container by executing `crc start`. This will take a few minutes to complete, and the output should look something like this:
+[source,bash]
+----
+# Cluster monitoring is required for the tests to pass
+crc config set enable-cluster-monitoring true
 
-image::crc_start_output.png[align=center]
+# A minimum of 14GB of virtual machine memory is required to run by default, so it is recommended to change that
+# setting. Tweaking the CPUs and disk size is optional, but also recommended to avoid having issues down the line.
+crc config set cpus 8
+crc config set disk-size 50
+crc config set memory 14500 # or 20000 for 20GB
+----
 
-Take note of the `oc login...` command shown in the output.
+Now you can go ahead and start the cluster. The first time you will need to provide the pull secret you were presented
+with in the CRC download page — if you didn't grab it you can come back and copy it. Run the following command to start
+the cluster.
 
-8. Execute the following command to set up the necessary environment variables:
+[source,bash]
+----
+crc start
+----
 
-`eval $(crc oc-env)`
+While CRC boots, you can go ahead and prepare the Quay repositories and the local ones to be able to run the tests.
 
-9. Log into the cluster by executing the `oc login` command as shown in the resulting output from running `crc start` in the previous step.  The output should look something like this:
-+
-image::crc_oc_login.png[align=center]
+=== Prepare the projects
+==== Creating the Quay repositories and making them public
 
-10. Check out the CodeReady Toolchain `host-operator` and `member-operator` repositories and ensure that they are located at the same directory level as the `toolchain-e2e` repository.
-+
-* https://github.com/codeready-toolchain/host-operator
-* https://github.com/codeready-toolchain/member-operator
-+
-11. Change into the `toolchain-e2e` directory, and execute the tests.  It is recommended that you direct console output to a text file, as the test output is of significant length and is likely to be longer than your console buffer:
-+
-`make test-e2e-local >testoutput.txt`:
-+
+The end-to-end local tests build and push images to the Quay registry, so in order to avoid having issues with your
+CRC cluster being unable to pull them from a non-public repository, you need to create those repositories and make them
+"public". Follow the following steps for that:
 
+1. Log in to https://quay.io.
+2. Go to the repository section, or click on the following link: https://quay.io/repository.
+3. Click on the "Create new repository" button.
```

Comment:
Alternatively, you can run the e2e makefile targets first. It will create all the repos for you automatically when pushing the built images. Then you can go and change the created repos to public in quay.io. In this case you don't have to pre-create the repos.